test/mpi/impls/mpich/threads/testlist.in

```diff
@@ -0,0 +1 @@
+pt2pt
```

Comment:
Why does this need go through configure?

Reply:
Moved on non-contigure.